src/Certificate/Exception/AzureKeyVaultCertificateLocatorException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * This file is part of itk-dev/serviceplatformen.
+ *
+ * (c) 2020 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace ItkDev\Serviceplatformen\Certificate\Exception;
+
+class AzureKeyVaultCertificateLocatorException extends CertificateLocatorException
```

Comment:
Maybe a class comment about what an `LocatorException` is?

Reply:
This is the same for the other exception classes in this PR.

Reply:
Fixed in 4c835a3